### DISCUSSION TITLE
Which extension do I install to use notebooks?

### DISCUSSION THREAD

**Discussion by a4ginatl** [codefori/vscode-ibmi#1682] [General]
I saw the presentation that Liam Allen did on Common recently. I like this extension and want to install it. When I searched for it, I found a few notebooks and could not identify the one to install.

Which Notebook should I install?

TIA Darryl Freinkel.

**Comment by worksofliam** [comment]
Hi @a4ginatl - IBM i Notebooks will be part of the Db2 for i extension, but the feature has not yet been released. I will post back here when it has.

**Reply by staplJason** [reply]
New install and noticed the repo was archived.  Glad to hear it is not dead -- I really like this extension.
